lib/artsy-eventservice/artsy/event_service/rabbitmq_connection.rb

```diff
@@ -18,6 +18,7 @@ def self.build_connection
       def self.get_connection
         @mutex.synchronize do
           @connection ||= self.build_connection
+          @connection = self.build_connection if @connection.closed?
```

Comment:
was this biting us in some manifest way?

Reply:
Yes - you had a Sentry error pulled up but I can't find it now.  In any case trying to access the connection threw a `ConnectionClosed` exception.